kubejs/server_scripts/tags/greg_vein_tags.js

```diff
@@ -173,7 +173,7 @@ let gregVeinz = [
     ]
   },
   {
-    name: "diamond",
```

Comment:
now diamond is missing :face_with_spiral_eyes: 
get from here
https://github.com/AllTheMods/Gravitas2/blob/0ff9168ea8efaf9544ea139d0cff4e619d737d9a/kubejs/server_scripts/datapack/greg_vein_maker.js#L188-L199

@thevortex if globals were working at the start of the project... :roll_eyes: